Tests/SWBCoreTests/PIFLoadingTests.swift

```diff
@@ -683,14 +683,14 @@ private final class ProjectModelItemClass: ProjectModelItem {
         #expect(fileGroup.children.count == 2)
 
         // Examine its children
-        if let fileRef = try? #require(fileGroup.children.first as? FileReference) {
```

Comment:
We keep having to flip these #requires back and forth due to compiler bugs, I don't know that there's a way to avoid the warnings on every platform

Reply:
Is it a platform thing or a compiler version thing? I'm building with a relatively recent compiler.

Reply:
As I understand it it's a compiler version thing: https://github.com/swiftlang/swift/issues/79202